node/Cargo.toml

```diff
@@ -32,7 +32,9 @@ sc-transaction-pool = { workspace = true }
 sc-transaction-pool-api = { workspace = true }
 sc-offchain = { workspace = true }
 sc-consensus-aura = { workspace = true }
+sc-partner-chains-consensus-aura = { workspace = true }
 sp-consensus-aura = { workspace = true }
```

Comment:
Do we need to depend on the vanilla aura?

Reply:
Yes, see PR description.

Reply:
It is one of choices. The other way would be to copy-paste everything from our forked `sc-consensus-aura`.